src/main/java/com/github/manolo8/darkbot/config/Config.java

```diff
@@ -239,6 +232,10 @@
             public @Option @Number(max = 300, step = 1) int TRAIL_LENGTH = 15;
             public @Option boolean MAP_START_STOP = false;
             public @Option("colors") ColorScheme cs = new ColorScheme();
+            public @Option boolean CUSTOM_BACKGROUND = false;
+            public @Option @Percentage double TRANSPARENCY = 0.3f;
+            public @Option ImageWrapper CHOOSER = new ImageWrapper();
```

Comment:
I'd personally bundle these inside a sub-confid, that will make ti clearer that "transparency" is also about the image

Custom Background: 
    - Enabled
    - Opacity
    - Image